src/api/app/views/webui2/webui/staging/workflows/_problems.html.haml

```diff
@@ -2,20 +2,31 @@
   max_problems = 5
   project_class = staging_project.name.tr(':', '_')
   item_project_class = 'hidden-item-' + project_class
+  left_problems = max_problems
+  failed_status_checks = staging_project.failed_status_checks
+  build_problems = staging_project.problems
+  total_length = failed_status_checks.length + build_problems.length
 
-- if staging_project.problems.empty?
+- if build_problems.empty? && failed_status_checks.empty?
   .text-center
     %i.fas.fa-check-circle.text-success
 - else
   %ul.list-group.list-group-flush
-    = render partial: 'problems_item', locals: { staging_project: staging_project, index: 0...max_problems }
-
-    - length = staging_project.problems.length
-    - if length > max_problems
+    - failed_status_checks[0...max_problems].each do |check|
```

Comment:
do we want to render openQA problems first? :thinking:

Reply:
@Ana06 not sure, at least in the current obs_factory staging they are shown first, so i took it over